src/functionaltests/java/com/ericsson/ei/files/DownloadFilesTestSteps.java

```diff
@@ -0,0 +1,110 @@
+package com.ericsson.ei.files;
+
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import java.io.File;
+
+import org.apache.commons.io.FileUtils;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Ignore;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+import com.ericsson.ei.subscriptions.query.QueryAggregatedObjectsTestSteps;
+import com.ericsson.ei.utils.FunctionalTestBase;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+
+@Ignore
+@AutoConfigureMockMvc
+public class DownloadFilesTestSteps extends FunctionalTestBase {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(QueryAggregatedObjectsTestSteps.class);
+    
+    private static final String subscriptionTemplateFilePath = "src/main/resources/templates/subscriptionsTemplate.json";
```

Comment:
In the previous tests those private static final variables has been named with uppercase letters and _ as seperator for words. Would be good if this followed that to.